README.md

```diff
@@ -1,4 +1,95 @@
 This project is an OpenWhisk package that allows you to communicate with Kafka or IBM Message Hub instances.
 
 ## Status
-This is still very much a work-in-progress. So far a feed provider service and feed action have been implemented. These allow you to create a trigger that fires when new messages are posted to the Kafka instance/topic provided. While this technically works, the current state is still "alpha" quality, and should only be used for kicking the tires.
+This project is currently "beta". Most of the major work is done, and now we are putting on the final touches to get this ready for production use.
+
+## Usage
+This package allows you to create triggers that react when messages are posted to either an IBM Message Hub instance, or to a generic kafka instance. Since the parameters required for each of these situations are different, there are two separate feeds to handle them: `/messaging/messageHubFeed` and `messaging/kafkaFeed`
+
+
+### Creating a Trigger that Listens to an IBM MessageHub Instance
+In order to create a trigger that reacts when messages are posted to a Message Hub instance, you need to use the feed named `messaging/messageHubFeed`. This feed supports the following parameters
+
+|Name|Type|Description|
+|---|---|---|
+|kafka_brokers_sasl|JSON Array of Strings|This parameter is an array of `<host>:<port>` strings which comprise the brokers in your Message Hub instance|
+|user|String|Your Message Hub user name|
+|password|String|Your Message Hub password|
+|topic|String|The topic you would like the trigger to listen to|
+|kafka_admin_url|URL String|The URL of the Message Hub admin REST interface|
+|api_key|String|Your Message Hub API key|
+|isJSONData|Boolean (Optional - default=false)|When set to `true` this will cause the feed to attempt the message content as JSON before passing it along as the trigger payload.|
+
+While this list of parameters may seem daunting, they can be automatically set for you by using the `package refresh` CLI command.
+
```

Comment:
Give me an example to create a trigger using the Message Hub feed like you did for Kafka.